src/server/README.md

```diff
@@ -380,3 +380,40 @@ Faculty Member API
             -   errors:
                 -   Missing fields in body -> Status code: 400
                 -   Internal server error -> Status code: 500
+
+Student API
+
+-   Interacts with:
+
+    -   User and Student tables
+
+-   Routes:
+
+    -   api/studentProfile/update
+
+        -   Updates the fields of existing student and associated User object from the database
```

Comment:
Updates the fields of an existing student and associated user object from the database

Reply:
Done.